CMakeLists.txt

```diff
@@ -52,7 +52,7 @@ if(NOT CMAKE_C_STANDARD)
 endif()
 
 if(NOT CMAKE_CXX_STANDARD)
-    set(CMAKE_CXX_STANDARD 11)
+    set(CMAKE_CXX_STANDARD 17)
```

Comment:
My thoughts:
- Could this be limited to yocto? I think we're targeting to make OTBR work on C++11.
- Could we specify an older version of protobuf version so it doesn't have to depend on `std::string_view`? cc @morningboata

Reply:
Currently the protobuf is install by the system's libprotobuf-dev and protobuf-compiler library. It's hard to config the custom proto generator & library, and change the CMAKER's default proto generator & library MACROs. I don't know what is the proper way yet.

Reply:
@Deepan-Shiva Could you set the `CMAKE_CXX_STANDARD` to 17 in yocto-specific build configuration?